bin/here.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
As discussed, working from within a container can be a limitation with some experiments, but a `venv` might be a good thing for installing `faasmcli` and `faasmtools` both necessary to inherit tasks from `faasm`.

Reply:
There's a script in the Faasm repo to set up a venv with `faasmtools`: https://github.com/faasm/faasm/blob/master/bin/create_venv.sh 

Could either copy that or call it.

Reply:
I'd need to adapt the script to run against the adequate `venv` (note the `${PROJ_ROOT}` definition). And I think the marginal gain is limited (in terms of LoC and complexity).